src/knoop/Node.hpp

```diff
@@ -25,6 +25,10 @@ class Node {
   #include "knoop/Node/src/push_front.hpp"
   #include "knoop/Node/src/pop_back.hpp"
   #include "knoop/Node/src/pop_front.hpp"
+  #include "knoop/Node/src/front.hpp"
```

Comment:
These headers should probably be in the repository

Reply:
`<facepalm>`